.github/workflows/release-drafter-product.yml

```
@@ -0,0 +1,36 @@
+name-template: "v{{next_release.version}}"
```

actionlint (1.7.7)

unexpected key "name-template" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)